Gemfile.lock

```diff
@@ -1,28 +1,18 @@
 PATH
   remote: .
   specs:
-    mina-circle (1.2.1)
+    mina-circle (1.3)
```

Comment:
Yeah, I realized that when I bumped the version on #35. I'll fix it tonight-ish.